deploy/charts/harvester/templates/cattle-system.yaml

```diff
@@ -0,0 +1,24 @@
+# system-upgrade-controller watches secret and node change.
+# If there is a new node, system-upgrade-controller creates a job on the node to change CA certificates.
+# If additional-ca is changed, system-upgrade-controller creates a job on each node to change CA certificates.
+apiVersion: upgrade.cattle.io/v1
+kind: Plan
+metadata:
+  name: sync-additional-ca
+  namespace: cattle-system
+spec:
+  concurrency: 1
+  nodeSelector:
+    matchLabels:
+      harvesterhci.io/managed: "true"
+  serviceAccountName: system-upgrade-controller
+  secrets:
+    - name: harvester-additional-ca
+      path: /ca
+  version: v1.0.2
+  upgrade:
+    image: alpine:3.12
```

Comment:
Should we continue using apline? I thought we are in the transition to BCI images or else

Reply:
@johnliu55tw @bk201 do you have any recommended image to run the script here? Thank you.

Reply:
I'd recommend BCI image and better template it from values.yaml file.

Reply:
Updated it. Thank you.